lib/rspec/core/example_group.rb

```diff
@@ -487,14 +489,22 @@ def self.store_before_context_ivars(example_group_instance)
       end
 
       # @private
+      def self.currently_executing_a_context_hook?
+        @currently_executing_a_context_hook
+      end
```

Comment:
We previously discussed getting rspec-rails onto only public APIs. I still think this is a worth while goal. Any reason this should be `private`?